cl/corpus_importer/management/commands/scrape_pacer_free_opinions.py

```diff
@@ -16,6 +16,9 @@
 from requests import RequestException
 from urllib3.exceptions import ReadTimeoutError
 
+from cl.corpus_importer.management.commands.bulk_iquery_project import (
+    CycleChecker,
+)
```

Comment:
Let's move this to a better place, where we have utilities of some kind. I think we're going to want this feature in more and more of our scripts.

Reply:
I'd say keep it simple for now?